Mathlib/LinearAlgebra/Matrix/ToLinearEquiv.lean

```diff
@@ -200,39 +196,4 @@ alias nondegenerate_iff_det_ne_zero ↔ Nondegenerate.det_ne_zero Nondegenerate.
 
 end Nondegenerate
 
-end LinearEquiv
-
-section Determinant
-
-open BigOperators
-
-lemma det_ne_zero_of_neg [DecidableEq n] {S : Type _} [LinearOrderedCommRing S]
```

Comment:
This version has the nice property of working for `Int`

Reply:
Well, you still can prove the result for `Int` using the new version if you first cast to reals but I agree that it is less convenient to use in this case.

Reply:
Is this provable fully from your statement? I don't think there's any reason to delete this if it's not strictly stronger